hugegraph-llm/src/tests/middleware/test_middleware.py

```
@@ -0,0 +1,85 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from fastapi import FastAPI, Request, Response
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import

(F401)